.gitmodules

```diff
@@ -18,7 +18,10 @@
 	url = https://github.com/meta-rust/meta-rust.git
 [submodule "layers/meta-up-board"]
 	path = layers/meta-up-board
-	url = https://github.com/AaeonCM/meta-up-board.git
+	url = https://github.com/up-division/meta-up-board
```

Comment:
I have asked AAEON if this is the newer official BSP and I am waiting for a reply from them

Reply:
I've allready asked Daniel *** from AAEON in February and his answer is the reason for this PR.

```
We have already released, but changed repository.

From now on please refer always to this one, which now include Kirkstone: https://github.com/up-division/meta-up-board

Please make sure you follow the instructions in the readme carefully and let me know if you find any issue (the release is still in beta).
As the recipe integration with the new up pinctrl driver is not complete, you need to download it manually with git after cloning the meta-up-board repository (it is described in the readme).
```

Reply:
I'd like a word from them if possible

Reply:
Sure, I can forward you the email conversation if you like.